linkerd/core/src/main/scala/io/buoyant/linkerd/package.scala

```diff
@@ -39,6 +39,18 @@ import com.twitter.finagle.Stack
  * discovered at runtime with finagle's `LoadService` facility.
  */
 package object linkerd {
+  /**
+   * Reimport [[com.twitter.finagle.buoyant.ParamsMaybeWith ParamsMaybeWith]]
+   * so that it's globally visible in the `io.buoyant.linkerd` package.
+   * @param params a [[com.twitter.finagle.Stack.Params Stack.Params]] to
+   *               implicitly enhance with the `maybeWith()` function
+   * @return `params` enhanced with the
+   *        [[ParamsMaybeWith.maybeWith() maybeWith()]] function
+   */
+  @inline
+  final implicit def paramsMaybeWith(params: Stack.Params): ParamsMaybeWith =
+    ParamsMaybeWith(params)
+
   implicit class MaybeTransform[A](val a: A) extends AnyVal {
```

Comment:
Moved in 4b90419fa1b953ed483433ed353b49488c983584, but a quick code search indicates that we never seem to actually use `maybeTransform` anywhere?